docs/dapp/sapphire/quickstart.mdx

```diff
@@ -154,8 +154,8 @@ Pretty cool if you ask me!
 
 Anyway, make it happen by running
 
-```sh
-PRIVATE_KEY="0x..." pnpm hardhat run scripts/run-vigil.ts --network sapphire_testnet
+```shell
```

Comment:
Oof. We are not 100% consistent in this repo. 🤷‍♂️❓

Reply:
Good call, I made those consistent now.